src/shared/components/dataFormItem/dataFormItem.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Label from 'shared/components/label/label';
+import styles from './dataFormItem.css';
+
+class DataFormItem extends Component {
+
+  render() {
+    return (
+      <div className={styles.dataFormItem}>
+        {this.props.label && <Label htmlFor={this.props.id}>{this.props.displayName}</Label>}
+        <input id={this.props.id} type={this.props.type} value={this.props.value} />
```

Comment:
This should probably have a prop that controls autocomplete as well. Is autocomplete on for password fields?